### PR TITLE
feature/US036-Deploy_API_to_AWS

### PR DESCRIPTION
**Pull request by IamGalexing**

### CODE REVIEW

File: .github/workflows/aws.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: AWS DEPLOYMENT
+
+on:
+  push:
+    branches: ["develop", "master"]
+  pull_request:
+    branches: ["develop", "master"]
```
Comment (ljercha) [inline .github/workflows/aws.yml]:
```suggestion
    branches: ["dev"]
  pull_request:
    branches: ["dev"]
```

File: .github/workflows/aws.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build & push server
+        working-directory: "./server"
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: team-andrzejki
+          IMAGE_TAG: andrzejki-api
```
Comment (ljercha) [inline .github/workflows/aws.yml]:
```suggestion
          ECR_REPOSITORY: team-sor
          IMAGE_TAG: sor-api
```

File: .github/workflows/aws.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-1
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build & push server
+        working-directory: "./server"
```
Comment (ljercha) [inline .github/workflows/aws.yml]:
```suggestion
        working-directory: "./product-backend"
```

File: .github/workflows/aws.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: team-andrzejki
+          IMAGE_TAG: andrzejki-api
+        run: |
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+
+      - name: Build & push client
+        working-directory: "./client"
```
Comment (ljercha) [inline .github/workflows/aws.yml]:
```suggestion
        working-directory: "./product-web-app"
```

File: .github/workflows/aws.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+name: AWS DEPLOYMENT
+
+on:
+  push:
+    branches: ["dev"]
+  pull_request:
+    branches: ["dev"]
```
Comment (leszekgu) [inline .github/workflows/aws.yml]: Before merge, this can be removed
```suggestion
```

**Review by leszekgu — APPROVED** [review]

**Pull request merged**